src/constants_and_utils/utils.js

```diff
@@ -17,8 +19,46 @@ function waitForElementToBeClickable(htmlElement, timeoutInMilliseconds, errorMe
   browser.wait(EC.elementToBeClickable(htmlElement), timeoutInMilliseconds, errorMessage);
 }
 
+function protractorHelperWarning() {
+  return colors.yellow("Protractor-helper warning: ");
```

Comment:
I prefer using protractor-helper all lowercase.

Reply:
I thing with the 'P' in uppercase is more easy to identify the phrase start.

Reply:
I prefer to stick with the original name of the library, all lowercase. It's like call appear.in Appear.in, which is incorrect.